src/commands/local.ts

```diff
@@ -1051,6 +1055,49 @@ function startCronJobs(
         }
     }
 
+    if (yamlProjectConfiguration.services && yamlProjectConfiguration.services.crons) {
+        for (const cronService of yamlProjectConfiguration.services.crons) {
+            const functionName = await evaluateResource(
+                yamlProjectConfiguration,
+                cronService.function,
+                "local",
+                undefined,
+                {
+                    isLocal: true,
+                    port: port,
+                },
+            );
+            const endpoint = cronService.endpoint?.replace(/^\//, "");
+            const cronString = cronService.schedule;
+            const baseURL = `http://localhost:${port}/.functions/function-${functionName}`;
```

Comment:
I think this is prone to error in the future. If we change the `genezio local` command implementation and we serve the functions from a different link, this is very likely to be broken. The developer who will do that is likely to forget to change also here.